SoftLayer/CLI/user/remove_access.py

```diff
@@ -0,0 +1,38 @@
+"""User remove access to devices."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('identifier')
+@click.option('--hardware', '-h',
+              help="Display hardware this user has access to.")
+@click.option('--virtual', '-v',
+              help="Display virtual guests this user has access to.")
+@click.option('--dedicated', '-l',
+              help="dedicated host ID ")
+@environment.pass_env
+def cli(env, identifier, hardware, virtual, dedicated):
+    """User details."""
```

Comment:
Maybe it's necessary to add more details about the command, for example: USAGE and EXAMPLE